docs/element-definitions.md

```diff
@@ -16,32 +16,25 @@ The Base API Element contains four properties: `element`, `meta`, `attributes`,
 
 - `element` (string, required)
 
-    The `element` property defines the name of element. It MUST be a string that references an element, which SHOULD be defined.
+    The `element` property specifies the name of element. It MUST be a string that references an element, which SHOULD be defined.
 
-- `meta` (enum)
+- `meta`
 
-    The `meta` property is a reserved object for Refract-specific values. When `meta` is an object, it MAY contain elements itself. The element definition SHOULD be used when interacting with `meta` and its properties and values.
+    The `meta` property is a reserved object for Refract-specific values that MAY contain elements itself. The element definition SHOULD be used when interacting with `meta` and its properties and values.
 
-    - Members
-        - (object)
-            - `id` - Unique Identifier, MUST be unique throughout the document
-            - `ref` (Element Pointer) - Pointer to referenced element or type
-            - `classes` (array[string]) - Array of classifications for given element
-            - `title` (string) - Human-readable title of element
-            - `description` (string) - Human-readable description of element
-            - `links` (array[Link Element]) - Meta links for a given element
-        - (array[Member Element])
+    - `id` ([String Element][]) - Unique Identifier, MUST be unique throughout the document
+    - `ref` ([Ref Element][]) - Pointer to referenced element or type
+    - `classes` (Array Element[[String Element][]]) - Array of classifications for given element
+    - `title` ([String Element][]) - Human-readable title of element
+    - `description` ([String Element][]) - Human-readable description of element
+    - `links` (Array Element[[Link Element][]]) - Meta links for a given element
 
-- `attributes` (enum)
+- `attributes`
 
-    The `attributes` property defines attributes about the given instance of the element, as specified by the `element` property. When `attributes` is an object, it MAY contain elements itself. The element definition SHOULD be used when interacting with `attributes` and its properties and values.
+    The `attributes` property defines attributes about the given instance of the element, as specified by the `element` property. It is an object where properties are strings and their values are elements. The element definition SHOULD be used when interacting with `attributes` and its properties and values.
```

Comment:
Following on from https://github.com/apiaryio/api-elements/pull/22/files#r135813419, I don't think the last part of the sentence adds much and can be removed. The sentence before is clear and replaces it.

> The element definition SHOULD be used when interacting with `attributes` and its properties and values.

We already mention:

> It is an object where properties are strings and their values are elements.

Reply:
👍

Reply:
Updated.